apps/compounds_tester/lib/compounds_tester_web/components/compounds/accordion.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Compounds.Accordion do
+  @moduledoc """
+    A simple Accordion  component.
+  """
+
+  use Phoenix.Component
+  alias Phoenix.LiveView.JS
+
+  attr :class, :string, default: nil
+  attr :title, :string, required: true
+  attr :shadow, :boolean, default: false
+  attr :expanded?, :boolean, default: false
+  attr :id, :string, default: Compounds.Id.generate("contents")
+  slot :subtitle, required: false
+
+  slot :contents, required: true do
```

Comment:
Should this be named "items" for clarity?

Reply:
"Items" to me implies some sort of list or varying number of things where as "content" can be anything, which is the case here.